public/calendar.html

```diff
@@ -5,8 +5,145 @@
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>Calendar Page</title>
+		<link rel="stylesheet" href="../src/css/calendar.css" />
+		<link
+			rel="stylesheet"
+			href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+		/>
 	</head>
 	<body>
-		<h1>This is Calendar Views</h1>
+		<header>
+			<form>
+				<div class="navigation">
+					<div class="navigation__search-wrapper">
+						<input
+							type="search"
+							class="navigation__search-input"
+							placeholder="Search..."
+						/>
+						<button class="navigation__search-button" type="submit">
+							<i class="fa fa-search search-icon fa-lg"></i>
+						</button>
+					</div>
+					<div class="navigation__links">
+						<ul class="navigation__list">
+							<li class="navigation__list-style">
+								<a
+									class="navigation__list-link"
+									href="http://127.0.0.1:5500/public/index.html?"
+									>Homepage</a
+								>
+							</li>
+							<li class="navigation__list-style">
+								<a
+									class="navigation__list-link"
+									href="http://127.0.0.1:5500/public/calendar.html"
+									target="_blank"
```

Comment:
You should remove target="_blank" because you don't need to open same window in another tab.